sdk/src/main/java/io/opentdf/platform/sdk/Assertion.java

```diff
@@ -0,0 +1,107 @@
+package io.opentdf.platform.sdk;
+
+public class Assertion {
+
+    public enum Type {
+        HandlingAssertion("Handling"),
+        BaseAssertion("Base");
+
+        private String type;
+
+        Type(String assertionType) {
+            this.type = assertionType;
+        }
+
+        @Override
+        public String toString() {
+            return type;
+        }
+    }
+
+    public enum Scope {
+        TrustedDataObj("TDO"),
+        Payload("PAYL"),
+        Explicit("EXPLICIT");
+
+        private String scope;
+
+        Scope(String scope) {
+            this.scope = scope;
+        }
+
+        @Override
+        public String toString() {
+            return scope;
+        }
+    }
+
+    public enum AppliesToState {
+        Encrypted("encrypted"),
+        Unencrypted("unencrypted");
+
+        private String state;
+
+        AppliesToState(String state) {
+            this.state = state;
+        }
+
+        @Override
+        public String toString() {
+            return state;
+        }
+    }
+
+    public enum StatementFormat {
+        ReferenceStatement("ReferenceStatement"),
+        StructuredStatement("StructuredStatement"),
+        StringStatement("StringStatement"),
+        Base64BinaryStatement("Base64BinaryStatement"),
+        XMLBase64("XMLBase64"),
+        HandlingStatement("HandlingStatement"),
+        StringType("String");
```

Comment:
Maybe we should remove these for now since they don't match the examples in the spec (e.g. json-structured)  and until we have a use case to marshal/unmarshal these examples.